tests/django_s3_storage_test/settings.py

```diff
@@ -16,28 +16,17 @@
 
 AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY")
 
-AWS_REGION = os.environ.get("AWS_REGION", "us-east-1")
-
-AWS_S3_BUCKET_NAME = os.environ.get("AWS_S3_BUCKET_NAME")
+AWS_REGION = os.environ.get("AWS_REGION", "eu-central-1")
 
 AWS_S3_KEY_PREFIX = uuid.uuid4().hex
 
-AWS_S3_BUCKET_NAME_STATIC = os.environ.get("AWS_S3_BUCKET_NAME")
-
-AWS_S3_KEY_PREFIX_STATIC = uuid.uuid4().hex
-
 DEFAULT_FILE_STORAGE = "django_s3_storage.storage.S3Storage"
 
-STATICFILES_STORAGE = "django_s3_storage.storage.ManifestStaticS3Storage"
-
+DEBUG = True
```

Comment:
What implications does this have?

Reply:
It is useful during debugging (duh!) Django. More details on error pages and better traces on exceptions. But no impact on django-s3-storage itself. 

I don't think we needed anymore for now, so I will remove it.